enterprise/coderd/provisionerkeys.go

```diff
@@ -147,9 +147,11 @@ func (api *API) provisionerKeyDaemons(rw http.ResponseWriter, r *http.Request) {
 
 	pkDaemons := []codersdk.ProvisionerKeyDaemons{}
 	for _, key := range sdkKeys {
-		// currently we exclude user-auth from this list
+		// Currently the user-auth key orgID is hardcoded to the default org.
+		// This will be changed when we update the `user-auth` keys to be
+		// directly tied to a user ID.
 		if key.ID.String() == codersdk.ProvisionerKeyIDUserAuth {
-			continue
+			key.OrganizationID = organization.ID
```

Comment:
This says the default org, but the `organization` could be any org right? The `organization.ID` comes from the org in the url

Reply:
Comment is confusing and will update it. The `user-auth` `key.OrganizationID` is hardcoded to the `default` org in the database and we are overwriting it here. The `organization` is valid here because it's the ID we use to query the list to begin with. It's just that for provisioners in the non-default org that use `user-auth` would have the default org ID returned here, so we edit the fake `user-auth` key to have the correct org here before returning it.